packages/polywrap-client-config-builder/polywrap_client_config_builder/client_config_builder.py

```diff
@@ -0,0 +1,185 @@
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from polywrap_core import Uri, IUriResolver, Env
+from typing import Dict, Any, List, Optional, Union
+
+@dataclass(slots=True, kw_only=True) 
+class ClientConfig():
+    """
+    This Abstract class is used to configure the polywrap client before it executes a call
+    The ClientConfig class is created and modified with the ClientConfigBuilder module
+    """
+    envs: Dict[Uri, Dict[str, Any]]
+    interfaces: Dict[Uri, List[Uri]]
+    resolver: List[IUriResolver]
+    wrappers: List[Uri]
```

Comment:
```
export interface ClientConfig<TUri extends Uri | string = Uri | string> {
    readonly envs: Env<TUri>[];
    readonly interfaces: InterfaceImplementations<TUri>[];
    readonly wrappers: IUriWrapper<TUri>[];
    readonly packages: IUriPackage<TUri>[];
    readonly resolvers: UriResolverLike[];
}
```
Fix this according to js ccb

Reply:
Without redirects though

Reply:
i think it make sense to have redirects in builder configuration

Reply:
fwiw js client config still has redirects https://github.com/polywrap/toolchain/blob/origin-dev/packages/js/client-config-builder/src/ClientConfig.ts#L14